trio_parallel/_impl.py

```diff
@@ -207,33 +208,94 @@ def statistics(self):
         )
 
 
-# intentionally skip open_worker_context
 DEFAULT_CONTEXT = WorkerContext._create()
 DEFAULT_CONTEXT_RUNVAR = trio.lowlevel.RunVar("win32_ctx")
+DEFAULT_CONTEXT_PARAMS = {}
+
+
+def configure_default_context(
+    idle_timeout=DEFAULT_CONTEXT.idle_timeout,
+    init=DEFAULT_CONTEXT.init,
+    retire=DEFAULT_CONTEXT.retire,
+    grace_period=DEFAULT_CONTEXT.grace_period,
+    worker_type=WorkerType.SPAWN,
+):
+    """Configure the default worker cache parameters.
+
+    Args:
+      idle_timeout (float): The time in seconds an idle worker will
+          wait for a CPU-bound job before shutting down and releasing its own
+          resources. Pass `math.inf` to wait forever. MUST be non-negative.
+      init (Callable[[], bool]):
```

Comment:
it's just an ideal, but it would be nice if there was a WorkerContextConfig dataclass with public attributes that is accepted here and by the WorkerContext, and the attributes only need to be documented in one place.

Reply:
Agree the duplication is unfortunate. I'd consider a separate PR for this, if one happened to show up...